### DISCUSSION TITLE
New update: "Align to corner of screen" not working

### DISCUSSION THREAD

**Discussion by alan2675** [Open-Shell/Open-Shell-Menu#1660] [Q&A]
Windows 11 installed an update this morning.  I immediately got a notification that there is a new version of Open Shell (4.4.190), which I installed. Now the Start Button appears on top of the Windows start button and not in the corner of the screen.  That option is checked in Settings.  Is there a workaround I can use? Or will there be a fixed edition released soon?  Thanks!

**Comment by ge0rdi** [comment]
Please, see #1252 regarding start menu/button position.
